code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
```

Comment:
To be fair.. some of us use T to chat. *[emote here] tends to be easier. There isn't a /me bit, is there?